core/storage/certificate_assessment/gae_models.py

```
@@ -274,3 +276,265 @@ def create(
+        return dict(
+            super(cls, cls).get_export_policy(),
+            **{
+                'learner_id': base_models.EXPORT_POLICY.NOT_APPLICABLE,
+                'total_score': base_models.EXPORT_POLICY.EXPORTED,
+                'attempt_index': base_models.EXPORT_POLICY.EXPORTED,
+                'attempt_data': base_models.EXPORT_POLICY.EXPORTED,
+                'version_data': base_models.EXPORT_POLICY.NOT_APPLICABLE,
+                'started_at': base_models.EXPORT_POLICY.EXPORTED,
+                'finished_at': base_models.EXPORT_POLICY.EXPORTED,
+                'is_submitted': base_models.EXPORT_POLICY.EXPORTED,
+            },
```

Ruff (0.15.20)

[warning] Unnecessary `dict` kwargs

Remove unnecessary kwargs

(PIE804)